backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -47,22 +51,41 @@ private void updatePersonFacility(Person person, UUID facilityId) {
         person.setFacility(facility);
     }
 
-    @AuthorizationConfiguration.RequirePermissionSearchPatients
+    @AuthorizationConfiguration.RequirePermissionReadPatientList
     public List<Person> getPatients(UUID facilityId) {
-        Organization org = _os.getCurrentOrganization();
-        if (facilityId == null) {
-            return _repo.findAllByOrganization(org, NAME_SORT);
-        }
-        Facility facility = _os.getFacilityInCurrentOrg(facilityId);
-        return _repo.findByFacilityAndOrganization(facility, _os.getCurrentOrganization(), NAME_SORT);
+        return _repo.findByFacilityAndOrganization(_os.getFacilityInCurrentOrg(facilityId),
+                _os.getCurrentOrganization(), NAME_SORT);
+    }
+
+    @AuthorizationConfiguration.RequirePermissionReadPatientList
+    public List<Person> getAllPatients() {
+        return _repo.findAllByOrganization(_os.getCurrentOrganization(), NAME_SORT);
+    }
+
+    @AuthorizationConfiguration.RequirePermissionReadPatientDeletedList
+    public List<Person> getPatientsInclDeleted(UUID facilityId) {
+        return _repo.findByFacilityAndOrganizationIncludeDeleted(_os.getFacilityInCurrentOrg(facilityId),
+                _os.getCurrentOrganization(), NAME_SORT);
+    }
+
+    @AuthorizationConfiguration.RequirePermissionReadPatientDeletedList
+    public List<Person> getAllPatientsInclDeleted() {
+        return _repo.findAllByOrganizationIncludeDeleted(_os.getCurrentOrganization(), NAME_SORT);
+    }
+
+    @AuthorizationConfiguration.RequirePermissionDeletePatient
+    public Person getPatientCheckInclDelete(String id) {
+        return _repo.findByIDAndOrganizationIncludeDeleted(UUID.fromString(id), _os.getCurrentOrganization())
+                .orElseThrow(()->new IllegalGraphqlArgumentException("No patient with that ID was found"));
     }
 
     // NO PERMISSION CHECK (make sure the caller has one!)
-    public Person getPatient(String id) {
-        return getPatient(id, _os.getCurrentOrganization());
+    public Person getPatientNoPermissionsCheck(String id) {
+        return getPatientNoPermissionsCheck(id, _os.getCurrentOrganization());
     }
 
-    public Person getPatient(String id, Organization org) {
+    // NO PERMISSION CHECK (make sure the caller has one!)
+    public Person getPatientNoPermissionsCheck(String id, Organization org) {
```

Comment:
We should push the UUID conversion up to the graphql layer when we get a chance, but I don't think today is necessarily that day. (OTOH today is a day when we're changing all the method names that use the String ID, so perhaps?)